app/controllers/provider_interface/organisation_permissions_setup_controller.rb

```diff
@@ -63,8 +63,12 @@ def check
 
     def commit
       wizard = OrganisationPermissionsSetupWizard.new(organisation_permissions_wizard_store)
+      setup_relationships = wizard.relationships.select(&:setup_at)
```

Comment:
Maybe call this `already_set_up_relationships`? `setup` is a bit ambiguous as to whether the user has just done this, or someone else did it

Reply:
Sounds good, sorted